client/src/containers/App/App.js

```diff
@@ -2,12 +2,18 @@ import React from "react";
 import "./App.css";
 import AppContextProvider from "./AppContext";
 import UserStatus from "../UserStatus";
+import SmallContainer from "../../components/SmallContainer/SmallContainer";
+import SignIn from "../SignIn/SignIn";
 
 function App() {
   return (
     <div className="App">
       <AppContextProvider>
-        <UserStatus />
+        {/* <SmallContainer>
```

Comment:
Remove comments.

Reply:
Removed comments in commit [c9a0c3e](https://github.com/pesto-students/batch-8-CODERHOOD/commit/c9a0c3e9a98041bd76431987b3050799c848896c)